.kiro/specs/smile-zemi-crawler-line-notify/requirements.md

```
@@ -31,7 +52,7 @@ githubsecretsにログイン時のユーザ名とパスワードが登録され
 6. If ログインが失敗する, then the Smile Zemi Crawlerワークフロー shall エラー詳細をログに記録してワークフローを終了する
 7. The Smile Zemi Crawlerワークフロー shall セッションCookieを保持して認証状態を維持する
 
-### Requirement 3: ミッション数取得機能
+### Requirement 3: 学習日、ミッション数、勉強時間取得機能
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)